### PR TITLE
Separate page for Git Workflow content

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**
Moved Git workflow content from Git index to separate page.

### CODE REVIEW

File: content/docs/how-we-work/git/git-workflow.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Git Workflow
+weight: 4
+description: Git branching and merging are essential concepts for managing any software. By using branches, we can isolate different features or bug fixes, and merge them back into the main branch as soon as they are ready.
```
Comment (hussainweb) [inline content/docs/how-we-work/git/git-workflow.md:4]: @Yemaneberhan-Lemma, the description is inaccurate

File: content/docs/how-we-work/git/git-workflow.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+# Git Workflow
+
+Git branching and merging are essential concepts for managing any software. By using branches, we can isolate different features or bug fixes, and merge them back into the main branch as soon as they are ready.
```
Comment (hussainweb) [inline content/docs/how-we-work/git/git-workflow.md:9]: Here as well.

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/how-we-work/git/git-workflow.md (hunk 1 above)
Comment (Yemaneberhan-Lemma) [reply content/docs/how-we-work/git/git-workflow.md]: Fixed.

File: content/docs/how-we-work/git/git-workflow.md (hunk 2 above)
Comment (Yemaneberhan-Lemma) [reply content/docs/how-we-work/git/git-workflow.md]: Fixed

**Review by hussainweb — APPROVED** [review]

**Pull request merged**